MPF.Processors/Redumper.cs

```diff
@@ -151,10 +151,12 @@ public override void GenerateSubmissionInfo(SubmissionInfo info, string basePath
                 case RedumpSystem.IBMPCcompatible:
                 case RedumpSystem.RainbowDisc:
                 case RedumpSystem.SonyElectronicBook:
-                    info.CopyProtection!.SecuROMData = GetSecuROMData($"{basePath}.log") ?? string.Empty;
+                    info.CopyProtection!.SecuROMData = GetSecuROMData($"{basePath}.log", out SecuROMScheme secuROMScheme) ?? string.Empty;
+                    if (secuROMScheme == SecuROMScheme.Unknown)
+                        info.CommonDiscInfo!.Comments += "Warning: Incorrect SecuROM sector count" + Environment.NewLine;
```

Comment:
Looks like I was wrong, you can just use `=` in both of these places and not `+=`. I had to double check in the submission tool.